frontend/packages/pipelines-plugin/integration-tests/testData/task-creation-with-yaml/create-pipeline-with-task.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: new-pipeline-one
+  namespace: aut-pipelines
```

Comment:
I would suggest not to use namespace because if you test that is running in a different namespace, then this yaml will always create in `auto-pipelines` namespace. If we don't specify it then yaml editor will create in the current testing namespace.
```suggestion

```